### PR TITLE
Updated comments section

### PR DESCRIPTION
**Pull request by jrothrock**

### CODE REVIEW

File: Chapter Three/Chapter 3 - Fabulous Forms & Pleasant Pagination.md [hunk 1]
```diff
@@ -694,6 +698,17 @@ The above process seems complex for what seemed initially to be a simple feature
 We just needed to add our extra partial and some dynamic id's and data-attributes to make this simple flow work correctly!
 
 
+Now this ish is working, but the comments are still appending after the link! This is where our Clicked variable is gonna come into play:
```
Comment (benwalks) [inline Chapter Three/Chapter 3 - Fabulous Forms & Pleasant Pagination.md]: Typo on line 701. *ish

**Comment by benwalks** [comment]
This is really great Jack, thanks.  I've spotted one typo, but will happily merge once that's fixed.  This is a great addition to the chapter.

**Comment by jrothrock** [comment]
That word fits perfectly haha. I got you though

**Comment by benwalks** [comment]
I was 50/50 on whether it was intentional!  Want to leave it?  It'd be great to leave your touch on the book.

**Comment by jrothrock** [comment]
It's up to you man. It's your book

**Comment by benwalks** [comment]
Alright, I'll leave it as per the latest commits.  Thanks again for your help.

**Pull request merged**